Tests/ApodiniTests/ParameterMutabilityTests.swift

```diff
@@ -28,6 +28,29 @@ class ParameterMutabilityTests: ApodiniTests {
                     .joined(separator: separator)
         }
     }
+    
+    class StringClass: Codable {
+        var string: String
+        
+        init(string: String) {
+            self.string = string
+        }
+    }
+    
+    struct TestHandlerUsingClassType: Handler {
+        @Parameter
+        var name = StringClass(string: "Apodini")
```

Comment:
Would you mind adding a test case (or adding an assert to the test case) for a optional parameter with a default value.
As Parameters with optional Type are treated differently than parameters with a non optional type (and also the defaultValue closure is constructed differently) I would find a test case like described important.

Reply:
Yes, seems reasonable. I will add one.